main.go

```diff
@@ -393,6 +393,8 @@ func main() {
 	for _, b := range config.Bundles {
 		chBundle <- b
 	}
+	close(chBundle)
+	<-done
```

Comment:
Although I added `done <- struct{}{}` at L412, but I forgot to retrieve it. So it might proceed even if it doesn't be  ready.